cpu/atmega_common/Makefile.features

```diff
@@ -1,2 +1,4 @@
 FEATURES_PROVIDED += periph_eeprom
 FEATURES_PROVIDED += periph_pm
+
+FEATURES_PROVIDED += atmega_pcint0 atmega_pcint1
```

Comment:
The ATmega32U4 only as PCINT0. Also, this forces every newly added implementation of ATmega CPUs to also add PCINT right away. Maybe it is better to let each individual ATmega CPU define the list of features provided?

Reply:
That would by the way also allow to split up the code changes into different commits, e.g. first adding the implementation in one commit, that add the configuration for each CPU as separate commit, than a final commit adding the Arduino-Mega2560 board-specific configuration.

I think this would help people porting new ATmega CPUs to RIOT, as they could see in a commit adding the PCINT config for a CPU what exactly has to be done.

Reply:
FEATURES_PROVIDED does not add the module, right? It should shows that the features is available - correct me if I'm wrong here!

Reply:
That's correct